src/main/resources/OpenSky.conf

```diff
@@ -0,0 +1,5 @@
+osc {
+    apiurl = "https://opensky-network.org/api/states/all"
```

Comment:
Please, rewrite configuration keys from camelcase to other format, either "a.b" or "a-b"